src/components/elements/card.js

```diff
@@ -83,50 +120,65 @@ const IconContainer = styled.div`
     ${Header} {
         display: flex;
         align-items: center;
-        padding-left: 16px;
     }
 `
-
+const CardContentContainer = styled.div`
+    margin-left: 1.6rem;
+`
 const Content = ({ content }) => (
     <>
         {Array.isArray(content) ? (
             content.map(text => <CardContent key={text}>{text}</CardContent>)
         ) : (
-            <CardContent>{content}</CardContent>
-        )}
+                <CardContent>{content}</CardContent>
+            )}
     </>
 )
 
 Content.propTypes = {
     content: PropTypes.oneOfType([PropTypes.string, PropTypes.array]),
 }
 
-export const Card = ({ children, Icon, title, content, width, min_height, is_inline_icon }) => {
+export const Card = ({ children, Icon, title, content, width, min_height, is_inline_icon, cover_background, cover_content }) => {
     return (
         <CardWrapper width={width} min_height={min_height}>
             {!children && (
                 <>
                     {is_inline_icon ? (
                         <>
+                            <CardCover background_color={cover_background}>
+                                <div>
+                                    <h4>
+                                        {cover_content}
+                                    </h4>
+                                    <h4>
+                                        &gt;
```

Comment:
can please add comment on what symbol is this

Reply:
done